src/tree/ResolvableTreeItemBase.ts

```diff
@@ -53,7 +61,10 @@ export abstract class ResolvableTreeItemBase extends AzExtParentTreeItem impleme
 
             const resolver = this.getResolver();
 
-            await this.runWithTemporaryDescription(context, 'Loading...', async () => {
+            await this.runWithTemporaryDescription(context, {
+                description: loading,
+                skipRefresh: true
```

Comment:
```suggestion
                softRefresh: true
```

In the utils package it's `softRefresh` (which I think is a better name as it's still doing _some_ refresh)